This crate has licensing issues and is only used for a developer test.

--- a/src/firecracker/Cargo.toml
+++ b/src/firecracker/Cargo.toml
@@ -23,5 +23,4 @@
 vmm = { path = "../vmm" }
 
 [dev-dependencies]
-cargo_toml = "0.13.0"
 regex = "1.5.5"
--- a/src/firecracker/tests/verify_dependencies.rs
+++ b/src/firecracker/tests/verify_dependencies.rs
@@ -1,3 +1,4 @@
+#![cfg(not(test))]
 // Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
